src/Tools/DynamoAddInGenerator/DynamoAddInGenerator.csproj

```diff
@@ -37,7 +37,7 @@
   </PropertyGroup>
   <ItemGroup>
     <Reference Include="DynamoInstallDetective">
-      <HintPath>$(PACKAGESPATH)\DynamoVisualProgramming.Core\lib\net47\DynamoInstallDetective.dll</HintPath>
+      <HintPath>$(PACKAGESPATH)\DynamoVisualProgramming.Core\lib\net48\DynamoInstallDetective.dll</HintPath>
```

Comment:
@AndyDu1985 This is a mistake, it was missed when upgrading DynamoCore and cause DynamoRevit build with errors.